Dockerfile

```diff
@@ -28,16 +28,7 @@ ARG CRT_FILES=""
 # Set this to the ScanCode version to use.
```

Comment:
Commit message:

> While "Alpine is not in a supported release by OpenJDK" [3],
eclipse-temurin [4] (which supersedes the deprecated adoptopenjdk [5])
does offer an Alpine image [6]. However, prefer to use the slightly
larger "17-jdk-focal" image instead as that JDK image can also be easily
used to *run* ORT in order to supersede #4178, so building and running
ORT share the same image.

I agree with the change, but I understand your argumentation as follows:
You use the slightly larger "17-jdk-focal" image instead of using an alpine base image.
What I think you mean is that you use the JDK image instead of the JRE image which was used before.

Reply:
No, I indeed mean "I'm using an Ubuntu (Focal) instead of an Alpine image" (independently of JRE vs. JDK) although it's larger, because I can then use the same image for running ORT, so only a single Ubuntu image needs to be downloaded instead of Ubuntu + Alpine images.

Reply:
I.e., there would also be an Alpine *JDK* image (see `eclipse-temurin:17-jdk-alpine`), and this is the explanation why this is *not* used.

Reply:
I've reworded the commit message a bit to hopefully make that clearer.